### PR TITLE
[CLOSED] Improvements to the Lifebringers Ghostrole

### PR DESCRIPTION
**Pull request by zydras**
## About The Pull Request

This pull request is an expansion of the existing ghost role: Lifebringers. It adds their seed vault to Icebox, remaps the entire vault and reduces its spawn cost at roundstart, making it spawn more often.

## How This Contributes To The Skyrat Roleplay Experience

Lifebringers, also known as "The ghostrole Botany tutorial", always felt a bit underdeveloped to me, which led to it being generally unplayed most shifts it popped up. With a few efforts, I worked to improve the ghostrole so it attracts more players both for the Botany gameplay and for general roleplay.

Screenshot (658)

Mapping changes:
- The vault has been made symmetrical in structure, to make it take all the space it could need inside its 20x20 spawning bounds.
- Plumbing tools, construction materials, internals, winter coats and a toolbox have been added to the Left wing of the vault. This little area mimics a bit your average EVA storage, save that it's for Lavaland/Icemoon. Helps the Primal Podpersons not die too, should their vault be breached (Thank you gibtonite)
- The Central wing of the vault now contains a little kitchen with tools for food preparation (mostly intended for guests). The Pod spawners have been centered to the middle of the vault, with its exit leading directly to the central area: A small area surrounded by grass. To the right of the Pod spawners is a little seating area, should the hard, dedicated workers of the seed vault wish to take a breather around fruit salads. This area is meant to be a bit of a central crossroad, separated from the "coldness" of the rest of the steel vault.
- Most of the South wing - where the hydroponics trays are at - has been left untouched save for the relocation of a few items. For ease, both vendors have been moved to be inside the South wing.
- The Right wing of the vault is much more focused on being storage: Three crates of seeds and vending restocks; Two crates of beekeeping gear; Shears and Somatorays, and the air tanks.

## Changelog

:cl:
add: Added the Lifebringer Seed Vault to the Icebox ruins list to diversify the ghostrole pool.
expansion: The Seed Vault was entirely remapped to fit the ghostroles better and to increase their ability to interact with the rest of the server.
balance: Reduced the spawning cost of the Seed Vault from 10 to 7 to make it appear more frequently on lavaland.
config: Changed some config setting related to blacklisted ruins.
/:cl:

### CODE REVIEW

File: code/datums/ruins/lavaland.dm [hunk 1]
```diff
@@ -41,7 +41,7 @@
 	description = "The creators of these vaults were a highly advanced and benevolent race, and launched many into the stars, hoping to aid fledgling civilizations. \
 	However, all the inhabitants seem to do is grow drugs and guns."
 	suffix = "lavaland_surface_seed_vault.dmm"
-	cost = 10
+	cost = 5
```
Comment (tf-4) [inline code/datums/ruins/lavaland.dm]: _Doubled_ is a lot. Maybe put it to like 7 / 8

Comment (zydras) [reply code/datums/ruins/lavaland.dm]: Done and done!

**Comment by SpaceVampire** [comment]
You should include images of the remap in the PR body, and explain any important changes made to it.

**Comment by tf-4** [comment]
I will say I like the idea. This role sucks tbh so having some life breathed into it will be nice

**Comment by YakumoChen** [comment]
Don't really like the idea of them getting free air tanks. They're fairly discouraged from leaving at least until they can grow space tobacco or the like to outheal the oxygen damage from the enviroment and takes away one of the first goals of the job. 

The same goes for the lanterns, you're encouraged to grow glowshrooms or something in order be able to venture out more easily. Lifebringers are a unique role with unique, botany-solvable problems (which is the point of the role) and you're solving their problems with a bunch of free gear.

(I also don't like that you've taken away the aesthetic that it's supposed to be a seed-ship / seed vault and just made it a dome thing in the middle of nowhere but that's just griping)

**Pull request closed without merging**